OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -21,6 +21,10 @@
 using Robust.Shared.Serialization.Markdown.Mapping;
 
 namespace OpenDreamRuntime.Procs.Native {
+    /// <remarks>
+    /// Note that this proc container also includes global procs which are used to create some DM objects,
+    /// like filter(), matrix(), etc.
+    /// </remarks>
     static class DreamProcNativeRoot {
```

Comment:
Every comment I make answers a question that I actually asked myself. For this one, the question was

> In our file structure, do we put native root procs in the same file as all of its methods, like placing `matrix()` within `DreamProcNativeMatrix`, or do we just place them into DreamProcNativeRoot.cs?

It's a bit unintuitive since they are technically constructors. I think you even confused `matrix()` for `new/matrix()` during testing.